tests/test_chronos_tools.py

```diff
@@ -912,7 +917,7 @@ def test_format_chronos_job_dict_invalid_param(self):
             branch_dict={},
         )
         with raises(chronos_tools.InvalidChronosConfigError) as exc:
-            invalid_config.format_chronos_job_dict('', [])
+            invalid_config.format_chronos_job_dict('', [], {})
```

Comment:
Can you use kwargs here? I have no clue what all of these empty strings/lists/dicts correspond to offhand.